src/components/Modal/staking-hub/DockerRunCommandModal.tsx

```
@@ -58,58 +64,222 @@ export const DockerRunCommandModal = (props: DockerRunCommandModalProps) => {
+      <div style={{ fontSize: '15px' }}>
+        Running HOPRd using Docker compose:
+        <ol>
+          <li>Download compose folder</li>
+          <li>Edit hoprd.cfg.yaml</li>
+          <li>Edit .env file</li>
+        </ol>
+        <a
+          href="https://docs-hoprnet-org-git-experiment-ui-tweak-300-hoprnet.vercel.app/node/node-docker-compose"
+          target="_blank"
+          rel="noreferrer"
+          style={{ color: '#007bff', textDecoration: 'underline' }}
```

ESLint

[error] Expected a line break after this opening brace.

(object-curly-newline)